agent/taskresource/credentialspec/credentialspec_windows.go

```diff
@@ -394,8 +398,14 @@ func (cs *CredentialSpecResource) handleS3CredentialspecFile(originalCredentials
 	}
 
 	resourceBase := filepath.Base(parsedARN.Resource)
-	localCredSpecFilePath := fmt.Sprintf("%s/s3_%s_%s.json", cs.credentialSpecResourceLocation, cs.taskARN, resourceBase)
+	taskArnSplit := strings.SplitN(cs.taskARN, "/", -1)
```

Comment:
You can just use Split(cs.taskARN, "/"), it works as the same way.

Reply:
Idiomatic simplification -- using Split() is equivalent, but doesn't mean passing in a magic number.